docs/source/usage.rst

```diff
@@ -0,0 +1,364 @@
+====================
+Basic Usage Examples
+====================
+
+For the examples on this page, we assume you've :doc:`installed the bigchaindb_driver JavaScript package <quickstart>`,
+and you have determined the BigchainDB Root URL (issue: move this to general docs)
+of the node or cluster you want to connect to.
+
+This example guides you through creating and transferring an asset.
+We walk through the code explaining its use, some pieces are left out
+because they have no real use (e.g. defenition of global variable)
+*Full working code* can be found at the bottom of this document.
+The following code are just snippets.
+
+Getting Started
+---------------
+We begin by creating an object of BigchainDB driver:
+
+.. code-block:: js
+
+	const driver = require('bigchaindb-driver')
+
+Next, we define a constant containing the API path.
+
+.. code-block:: js
+
+	const API_PATH = 'http://localhost:9984/api/v1/'
+
+Create Connection With BigchainDB
+---------------------------------
+
+A simpel connection with BigchainDB can be established like this.
+
+.. code-block:: js
+
+	const conn = new driver.Connection(API_PATH)
+
+It is also possible to connect to a BigchainDB node of the IPDB test net.
+To do so, you need to pass the **app_id and app_key**. 
+
+.. code-block:: js 
+
+	let bdb = new driver.Connection('https://test.ipdb.io/api/v1/', { 
+		app_id: 'dgi829l9',
```

Comment:
I hope these are not your actual keys to use the test net. If so please remove them from this PR.

Reply:
No, fake keys! :)

Reply:
But correct lenght

Reply:
👍